Tests/LibCrypto/TestHKDF.cpp

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2023, stelar7 <dudedbz@gmail.com>
```

Comment:
not me doing this 😅

Reply:
Initially I borrowed a lot of code from TestPBKDF2, which you did write. However, by now barely anything remains ^^'

I'll leave you in, but also add myself (no idea how I forgot that part).